GUI/MetaDraw.xaml.cs

```diff
@@ -476,6 +476,7 @@ private void DrawAnnotatedBaseSequence(PsmFromTsv psm)
             {
                 int residue = ion.NeutralTheoreticalProduct.TerminusFragment.AminoAcidPosition;
                 string annotation = ion.NeutralTheoreticalProduct.ProductType + "" + ion.NeutralTheoreticalProduct.TerminusFragment.FragmentNumber;
+                Color color = psm.VariantCrossingIons.ContainsKey(ion) && psm.VariantCrossingIons[ion] ? Colors.Green : productTypeToColor[ion.NeutralTheoreticalProduct.ProductType];
```

Comment:
`&& psm.VariantCrossingIons[ion]` doesn't do anything

Reply:
It does check to see if the entry with the key matched ion crosses the variant site. The dictionary is poorly named because it contains all of the matched ions, those that do and those that do not cross the variant site.

Reply:
VariantCrossingIons a List now, so that expression is now `psm.VariantCrossingIons.Contains(ion)`.